config.js

```diff
@@ -91,6 +91,31 @@ var config = convict({
                 "params": {
                     "model": "whisper-1"
                 },
+            },
```

Comment:
This config should be moved to AJ Cortex - don't want AJ-specific stuff in the OS repo.